passwordgenerator.cpp

```diff
@@ -0,0 +1,49 @@
+#include "passwordgenerator.h"
+
+std::string PasswordGenerator::makePassword()
```

Comment:
Użycie słowa "make" kiedy klasa nazywa się "generator" jest nie spójne. Raczej lepiej byłoby użyć słowa "generate". Ponadto słowo Password w nazwie funkcji jest raczej zbędne. Raczej powinno być oczywiste, że PasswordGenerator, generuje hasło, a nie buduje dom :) Nie trzeba tego precyzować. To samo tyczy się wszystkich późniejszych funkcji. Powtórzenie słowa Password w każdej z nich tylko zaciemnia. Z kontekstu wiadomo, że właśnie na nim pracujemy.